ros2param/ros2param/api/__init__.py

```diff
@@ -13,23 +13,15 @@
 # limitations under the License.
 
 import sys
-import warnings
 
 from rcl_interfaces.msg import ParameterType
 import rclpy
-from rclpy.parameter import get_parameter_value as rclpy_get_parameter_value
 from rclpy.parameter import parameter_dict_from_yaml_file
 from rclpy.parameter import parameter_value_to_python
 from rclpy.parameter_client import AsyncParameterClient
 from ros2cli.node.direct import DirectNode
 
 
-def get_parameter_value(*, string_value):
```

Comment:
this is imported by `ros2component`, addressed with https://github.com/ros2/ros2cli/pull/866/commits/bc80ce8de31c4323650d5b17beba117977a16b98